common/app/dev/DevParametersHttpRequestHandler.scala

```diff
@@ -53,8 +53,9 @@ class DevParametersHttpRequestHandler(
     "pbjs_debug", // set to `true` to enable prebid debugging,
     "amzn_debug_mode", // set to `1` to enable A9 debugging
     "force-braze-message", // JSON encoded representation of "extras" data from Braze
-    "dcr",
+    "dcr", // force page to render in DCR
     "topics", // used for filtering the liveblog blocks
+    "apps", // force page to render in Apps format
```

Comment:
I wonder if we could model this out a little. `dcr` is the rendering platform for both of these, so having `apps`, at least for me, confuses things a little.

In this case - if we say `Apps` format - if format something we're taking forward in the domain? (arguably it's a little fiddly given that's a term we already use).

Thinking of alternatives:

## Extending the `dcr` param

Given that we will be sending this to DCR, maybe the value of DCR would be `web | apps` with web as default as to not break current functionality? So this would be `?dcr=apps`.

I imagine this should be coupled to the work @OllysCoding has been doing around routing in DCR e.g. `/{Endpoint}/{RenderingTarget}`...

It feels like using the current parameter would make it easier to change or remove later.

Adding URLs always feels longer term and hard to deprecate, especially when adding new parameters.

Reply:
> I've added amp to the dcr query parameter

I can't see this anywhere? Maybe I've missed a commit?

Reply:
Whoops, looks like I never pushed the commit